tox.ini

```diff
@@ -0,0 +1,45 @@
+[travis]
+python =
+    3.6: py36
+    3.7: py37
+    3.8: py38
+    pypy3: pypy3
+
+[travis:env]
+DJANGO =
+    1.11: django111
```

Comment:
not needed

Reply:
Django 1.11 is an LTS release that will still be supported by DSF until April 2020. So we'll be providing support for 1.11 for now.

Reply:
Is there an actual downside for supporting Django 1.11 LTS for 5 more months? The test matrix passes for it and everything should work smoothly as it is.

Reply:
Well since the support was dropped in 4.0.0 I've removed it from the matrix as well, April will be soon and no point in reintroducing support after a major release.